modules/doc/content/getting_started/installation/wsl.md

```diff
@@ -1,45 +1,84 @@
 Begin by performing the following external instructions:
 
-- Install [Ubuntu 20.04](https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71) from the Microsoft store
 - Install [VcXsrv](https://sourceforge.net/projects/vcxsrv/reviews/) (Only needed for Peacock)
 
 ## Launch VcXsrv
 
-Each time you reboot, or basically each time VcXsrv is *not* running, and you wish to use the graphical capabilities of MOOSE (Peacock), you should start VcXsrv before launching your WSL terminal.
+Each time you reboot, or basically each time VcXsrv is *not* running, and you wish to use the graphical
+capabilities of MOOSE (Peacock), you should start VcXsrv before launching your WSL terminal.
 
-We have found better performance instructing VcXsrv to use software rendering over native OpenGL. When launching VcXsrv, search for the option: "Native opengl", and un-check it. All other options can remain the default.
+When starting VcXsrv, options for the server can be adjusted. In general, the default options are
+adequate, except for the following:
 
-## Edit Hostname within WSL
+- Un-check "Native opengl". We have found better performance instructing VcXsrv to use software
+  rendering over native OpenGL.
```

Comment:
```suggestion
```

Removing this, since we found that newer versions of Windows and WSL2 work fine with this enabled.